src/test/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { FeeAmount, Pool, Route as RouteV3 } from '@uniswap/v3-sdk'
+
+// helper function to make amounts more readable
+export const amount = (raw: TemplateStringsArray) => (parseInt(raw[0]) * 1e6).toString()
+
+export const USDC = new Token(1, '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48', 6, 'USDC')
```

Comment:
these tokens should already exist in constants/tokens.ts.
Instead of recreating, could you alias them here?

Reply:
i think we have these new definitions on purpose actually - we want versions of the tokens without the `name` field set, because the v3 SDK doesn't set that field. I didn't write these lines, i just copied them over to this util file

Reply:
would it break tests to use the version with `name` set? 🤔 feels fine to use the existing ones as is

Reply:
yes, that causes the tests to fail